specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-08-01-preview/IncidentCreationRules.json

```diff
@@ -0,0 +1,640 @@
+      "x-ms-enum": {
+        "name": "IncidentCreationRulePropertyConditionSupportedProperty",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "AlertDisplayName",
+            "description": "The name of the alert."
+          },
+          {
+            "value": "AlertAnalyticRuleId",
+            "description": "The analytic rule id of the alert."
+          },
+          {
+            "value": "AlertProductName",
+            "description": "The name of the product of the alert."
+          },
+          {
+            "value": "AlertSeverity",
+            "description": "The severity of the alert."
+          },
+          {
+            "value": "AlertType",
+            "description": "The type of the alert."
+          }
+        ]
+      }
+    },
+    "IncidentCreationRulePropertyGroupingSupportedProperty": {
+      "description": "The property to evaluate by which we will group alerts.",
+      "enum": [
+        "Entities",
+        "CustomDetails",
+        "AlertProperties"
+      ],
+      "type": "string",
+      "example": "Entities",
+      "x-ms-enum": {
+        "name": "IncidentCreationRulePropertyGroupingSupportedProperty",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Entities",
+            "description": "Group alerts by identical entities."
+          },
+          {
+            "value": "CustomDetails",
+            "description": "Group alerts by custom details."
+          },
+          {
+            "value": "AlertProperties",
+            "description": "Group alerts by alert properties."
+          }
+        ]
+      }
+    },
+    "IncidentCreationRulePropertyConditionProperties": {
+      "type": "object",
+      "properties": {
+        "propertyName": {
+          "$ref": "#/definitions/IncidentCreationRulePropertyConditionSupportedProperty"
+        },
+        "operator": {
+          "$ref": "#/definitions/IncidentCreationRulePropertyConditionSupportedOperator"
+        },
+        "propertyValues": {
+          "maxItems": 50,
+          "minItems": 1,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "IncidentCreationRulesList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IncidentCreationRule"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "PropertyBasedGroupingProperties": {
+      "description": "Describes a incident creation rule grouping configuration that evaluates properties values",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/IncidentCreationRuleGroupingConfiguration"
+        }
+      ],
+      "properties": {
+        "groupingProperties": {
+          "maxItems": 5,
```

Comment:
>"maxItems": 5 [](http://example.com/codeflow?start=10&length=13)

What does this do? I'd expect to see it on an array, but I don't know what it means on an object.